src/main/scala/bank/persistence/CustomerProfileDAO.scala

```diff
@@ -0,0 +1,7 @@
+package bank.persistence
+
+trait CustomerProfileDAO {
+  def getCustomerProfileByUsername(username: String): Option[CustomerProfile]
+  def saveCustomerProfile(customerProfile: CustomerProfile): Unit
+  def truncate(): Unit
```

Comment:
make sure that this method is not executed in production

Reply:
done by extending CustomerProfileDAO in test package and implementing truncate() exclusively there